build-config/resources/Ballerina.toml

```diff
@@ -0,0 +1,35 @@
+[package]
+org = "ballerinax"
+name = "kafka"
+version = "@toml.version@"
+distribution = "slbeta3"
+
+[[platform.java11.dependency]]
+path = "../native/build/libs/kafka-native-@project.version@.jar"
+groupId = "org.ballerinalang.messaging.kafka"
+artifactId = "kafka"
+version = "@project.version@"
+
+
+[[platform.java11.dependency]]
+path = "./lib/kafka-clients-@kafka.version@.jar"
+groupId = "org.apache.kafka"
+artifactId = "kafka-clients"
+version = "@kafka.version@"
+
+
+[[platform.java11.dependency]]
+path = "./lib/kafka_2.12-@kafka.version@.jar"
+groupId = "org.apache.kafka"
+artifactId = "kafka_2.12"
+version = "@kafka.version@"
+
+
+[[platform.java11.dependency]]
+path = "./lib/avro-1.9.2.jar"
```

Comment:
I think we should handle the Avro version as same we handle the Kafka version, using the placeholders, and put the Avro version in the `gradle.properties` file.

Reply:
Changed in https://github.com/ballerina-platform/module-ballerinax-kafka/pull/280/commits/2d0a36ee6ac39bd2f6eec146a23726b1eea39abe